src/ManifestStrategy/ConfigureEnvManifestStrategy.php

```diff
@@ -90,6 +90,33 @@ public function apply(array $manifest, string $moduleName, InputOutputInterface
         return true;
     }
 
+    /**
+     * @param string $target
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     *
+     * @return bool
+     */
+    protected function isTargetExist(string $target, InputOutputInterface $inputOutput): bool
```

Comment:
thats not a valid method name

targetExists() IMO

Reply:
Fixed